docs/maui/converters/is-in-range-converter.md

```diff
@@ -0,0 +1,83 @@
+
+> [!NOTE]
+> Note that the either **both** the `TrueObject` and `FalseObject` should have a value defined or **neither** should.
+
+The `ConvertBack` method is not supported.
+
+[!INCLUDE [common converter properties](../includes/communitytoolkit-converter.md)]
+
+## Syntax
+
+### XAML
+
+The `IsInRangeConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.IsInRangeConverterPage">
+        <ResourceDictionary>
+            <x:String x:Key="MinValue">Hello</x:String>
+            <x:String x:Key="MaxValue">World</x:String>
+            <Color x:Key="TrueColor">Green</Color>
+            <Color x:Key="FalseColor">Red</Color>
+        </ResourceDictionary>
+
+    <Label BackgroundColor="{Binding MyValue, Converter={toolkit:IsInRangeConverter TrueObject={StaticResource TrueColor}, FalseObject={StaticResource FalseColor}, MinValue={StaticResource MinValue}, MaxValue={StaticResource MaxValue}}}" Text="The background of this label will be green if MyValue is between 'Hello' and 'World', otherwise red." />
+
+</ContentPage>
+```
+
+### C#
+
+The `IsInRangeConverter` can be used as follows in C#:
+
+```csharp
+
+class IsInRangeConverterPage : ContentPage
+{
+    public IsInRangeConverterPage()
+    {
+        Content = new Label()
+            .Text("The background of this label will be green if MyValue is between 'Hello' and 'World', otherwise red.")
+            .Bind(
+                Label.BackgroundColorProperty,
+                nameof(ViewModel.MyValue),
+                converter: new IsInRangeConverter
+                {
+                        TrueObject = Colors.Green,
+                        FalseObject = Colors.Red,
+                        MinValue = "Hello",
+                        MaxValue = "World"
+                });
+    }
+}
+```
+
+## Properties
+
+|Property  |Type  |Description  |
+|---------|---------|---------|
+| MinValue | `IComparable` | The minimum value for the compare range. |
+| MaxValue | `IComparable` | The maximum value for the compare range. |
+| FalseObject | `object` | The result to return if the comparison results in a `false` comparison. |
+| TrueObject | `object` | The result to return if the comparison results in a `true` comparison. |
+
+## Examples
+
+You can find an example of this converter in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Converters/IsInRangeConverterPage.xaml).
```

Comment:
Looks like the link should be updated to `https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Converters/IsInRangeConverterPage.cs`?
(xaml -> cs)

Reply:
Thanks @JoonghyunCho .  I love how diligent you are, and I appreciate it.  I've been rushing things, and when you rush you can make mistakes.  So, I apologise for taking more of your time on this that is necessary.  Hopefully, I've finished it fully this time.

Reply:
Thanks Jay.  Brandon re-wrote the whole sample from XAML to code-behind and approved the PR before I was even notified.  I forgot that he had done this, and missed the impact to the documentation.  I have updated accordingly.

Reply:
No worries :) Thanks for the update!